src/EnvironmentManagement/EnvironmentManager.py

```diff
@@ -326,6 +326,72 @@ def manage_removal_from_game_for(self,
             return True
         else:
             return False
+        
+    def manage_car_switch_for(self,
```

Comment:
After further deliberation I don't see the point of manage_car_switch_for(). It only adds one line of code to the functionality of __switch_player_cars. __switch_player_cars could just me made public and contain the self.update_staff_ui() right before its return True.

If there is a grander reason for the separation, please enlighten me.

Reply:
The main reason would be because the player removal does it in a similar way and I started my implementation from that function. Maybe it could prove useful for future extensions to the car switch but I will follow your lead here.

Reply:
Okay I understand now. This might be useful in the future, when trying to signal to the players that their cars have been switched, score etc.